grails-app/controllers/org/pih/warehouse/JsonController.groovy

```diff
@@ -1869,6 +1869,50 @@ class JsonController {
 
         render([fieldRemoved: fieldRemoved] as JSON)
     }
+
+    def getPriceHistory = {
```

Comment:
I eventually want to get rid of the JsonController as it was a terrible idea.

So let's add this to the OrderController.

Reply:
Or SupplierController if we've added that.

Reply:
@jmiranda Thats a good suggestion. Thanks, done!